Composer/packages/client/src/components/ManageLuis/__tests__/ManageLuis.test.tsx

```diff
@@ -0,0 +1,378 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { fireEvent, act } from '@botframework-composer/test-utils';
+import React from 'react';
+
+import { renderWithRecoil } from '../../../../__tests__/testUtils';
+import { ManageLuis } from '../ManageLuis';
+
+const serviceName = 'Language Understanding';
+const DOWN_ARROW = { keyCode: 40 };
+
+// token creds
+jest.mock('@azure/ms-rest-js');
+
+jest.mock('@azure/arm-resources', () => ({
```

Comment:
you can reuse these mocks instead of repeating them:
https://jestjs.io/docs/manual-mocks

Reply:
I moved most of these to the __mocks__ folder so they can be shared. Thanks for the tip! The items i did not move are either different for the different tests or not mocking node modules.